test/mocha.opts

```diff
@@ -1,4 +1,3 @@
 --compilers js:babel-register
 --require test/helpers/globals
 --require babel-polyfill
---reporter mocha-trumpet-reporter
```

Comment:
For the time being, I recommend taking the `--reporter` flag out of `mocha.opts` and passing it the CLI directly.